src/TestFramework/Codeception/CustomAutoloadFilePath.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright © 2018 Tobias Stadler
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\TestFramework\Codeception;
+
+use Codeception\Events;
```

Comment:
1. it will not work since `codeception/codeception` is in `require-dev` section and all files from this section are absent in PHAR distribution
2. we need to find a way to not use Codeception dependency at all. Infection should not depend on any test framework.

could you please explain what is the goal of this file, we will try to find another way to achieve the same goal

Reply:
I don't thin this is a problem, because the code path is only used when the test framework is codeception.

Reply:
This could work if this file won't be a part of Infection. Instead you could tell Codeception to include a file from the outside Infection. E.g. there's an `IncludeInterceptor` which [is added to the tested code at one point](https://github.com/infection/infection/blob/5bfd1be14bd3762a573717e118f6115a3426d77f/src/TestFramework/PhpSpec/Config/Builder/MutationConfigBuilder.php#L70). I'm not entirely sure you could do what you want there, but there's an opportunity.